analytics/project_tracker.go

```diff
@@ -0,0 +1,43 @@
+package analytics
+
+import "sync"
+
+type ProjectTracker struct {
+	earthfileOrg     string
+	earthfileProject string
+	cliOrg           string
+	cliProject       string
+	mutex            sync.Locker
+}
+
+var projectTracker = ProjectTracker{
+	mutex: &sync.Mutex{},
+}
```

Comment:
I think it's perfectly fine to use a `sync.Locker`, but just FYI, if you used `sync.Mutex` directly instead, you wouldn't have to allocate the mutex here - you could just declare `var projectTracker ProjectTracker`. The zero value of a `sync.Mutex` is a fully working mutex lock for this express purpose.

Reply:
That's good to know, Thanks!
I used the interface so I can mock it in the tests, even though Alex already commented that was an overkill.